notebooks/7_reshape_table_layout.ipynb

```diff
@@ -0,0 +1,1654 @@
+{
```

Comment:
<p>typo: "titanic date" should be "titanic data"</p><p> </p><p>(This same typo is also below.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>